FontAwesome.xcodeproj/project.pbxproj

```diff
@@ -162,6 +166,7 @@
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
```

Comment:
did these things get auto updated when you opened the project in xcode 14?

Reply:
Yeah. Which is weird. Let me try another thing

Reply:
No, after setting `CODE_SIGNING_ALLOWED`

Reply:
We didn't ignore `xcodeproj` in `.gitignore` that's why all these weird things are here :(

Reply:
this is the intended workflow. carthage cannot execute xcodegen before compiling so we:
1. Update XcodeGen's project.yml file
2. Execute XcodeGen to generate xcodeproj file
3. Check in all changes